src/hooks/hookResources.ts

```diff
@@ -364,6 +382,9 @@ export async function handleCardOrder(
                       card.addLink(
                         createSmartLink(requirement.name, requirement.appContext, contextRequest)
                       );
+                      if (patient && patient.resourceType === 'Patient') {
```

Comment:
Does this also need to be added on the two places below that we are adding smart links as well? When I tried to send the request it doesn't come through this path and the suggestions are not present in the cards returned. This is only when there is etasu record already for this patient like when the enrollment form has already been completed.